src/instruments/instrumentstypes.h

```diff
@@ -171,6 +180,37 @@ struct PartInstrument {
 
 using PartInstrumentList = QList<PartInstrument>;
 
+struct ScoreOrderGroup {
+    int index;
+    QString family;
+    QString section { QString() };
+    QString unsorted { QString() };
+    bool bracket = false;
+    bool showSystemMarkings = false;
+    bool barLineSpan = false;
+    bool thinBracket = false;
+};
+
+struct InstrumentOverwrite {
+    QString id;
+    QString name;
+};
+
+struct ScoreOrder {
+    int index;
+    QString id;
+    QString name;
+    QMap<QString, InstrumentOverwrite> instrumentMap;
+    QList<ScoreOrderGroup> groups;
+
+    bool isValid() { return !groups.empty(); }
+};
+
+struct PartInstrumentListScoreOrder {
```

Comment:
I would prefer to rename this to OrderedPartInstrumentList or to PartInstrumentListWithOrder. What do you think?

Reply:
The structure contains a list of `PartInstrument`s **and** a `ScoreOrder`. `OrderedPartInstrumentList` suggests the `PartInstruments` are ordered. That might be true but it ignores the fact the structure also contains the score order itself.
`PartInstrumentListAndOrder` also suggests an order which is not necessarily true. A better name should be then `PartInstrumentListAndScoreOrder` but personally I think it too long.
The best name might be `PassedFormData`. It make it clear it is data from the form but doesn't try to explain **what** data is passed.